packages/syft/src/syft/service/request/request.py

```diff
@@ -248,7 +288,7 @@ def accept_by_depositing_result(self, result: Any, force: bool = False):
         is_approved = change.approved
 
         permission_request = self.approve()
```

Comment:
Currenlty, we need to approve twice. This because, the first approval, approves the code request. And only once the code is approved, we can access output_policy and hence create change requests for policy and permission.